### PR TITLE
Comment out leftovers

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Binary test complains regarding left size_in_bytes, so have to be commented out.

### CODE REVIEW

File: vcd/resource_vcd_independent_disk_test.go [hunk 1]
```diff
@@ -34,14 +34,14 @@ func TestAccVcdIndependentDiskBasic(t *testing.T) {
 
 	params["FuncName"] = t.Name() + "-Compatibility"
 	configTextForCompatibility := templateFill(testAccCheckVcdIndependentDiskForCompatibility, params)
-	params["FuncName"] = t.Name()
-	configText := templateFill(testAccCheckVcdIndependentDiskBasic, params)
-	if vcdShortTest {
+	/*	params["FuncName"] = t.Name()
+		configText := templateFill(testAccCheckVcdIndependentDiskBasic, params)
+	*/if vcdShortTest {
```
Comment (lvirbalas) [inline vcd/resource_vcd_independent_disk_test.go]:
```suggestion
	*/
	if vcdShortTest {
```

File: vcd/resource_vcd_independent_disk_test.go [hunk 2]
```diff
@@ -175,3 +176,4 @@ resource "vcd_independent_disk" "{{.secondResourceName}}" {
   storage_profile = "{{.storageProfileName}}"
 }
 `
+*/
```
Comment (lvirbalas) [inline vcd/resource_vcd_independent_disk_test.go]: I don't understand, why not leave this test, but only comment out the size_in_bytes field?

Comment (vbauzysvmware) [reply vcd/resource_vcd_independent_disk_test.go]: cause it's duplicate

Comment (dataclouder) [reply vcd/resource_vcd_independent_disk_test.go]: It was skipped in the acceptance test but was still exported to the binary tests

**Pull request merged**